<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-0SJ3B8VU" failures="1" tests="1" name="testOnEdge" time="8.103" errors="0" timestamp="20 Aug 2021 12:57:36 GMT">
  <testcase classname="com.inetbanking.testCases.TC_LoginTest_001" name="LoginTest" time="3.1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTest_001.LoginTest(TC_LoginTest_001.java:31)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- LoginTest -->
</testsuite> <!-- testOnEdge -->
